src/c3chart/CommonND.js

```diff
@@ -133,6 +135,23 @@
             break;
         }
 
+        if(this.y2() instanceof Array && this.y2().length > 0){
+            var axesObj = {};
+            var y2arr = this.y2();
+            this._config.axis.y2 = {
+                show: y2arr.length > 0
+            };
+            if(this.columns().length > 1){
+                var cols = this.columns();
+                for(var i = 0;i<y2arr.length;i++){
+                    for(var j = 1;j<cols.length;j++){
+                        axesObj[cols[j]] = y2arr.indexOf(cols[j]) !== -1 ? "y2" : "y";
+                    }
+                }
+                this._config.data.axes = axesObj;
+            }
+        }
```

Comment:
@mlzummo Sorry, I mean changing the y2 publish param in the property editor.   This array is what determines what is associated with the second axis.  In the test data that is [c2, c3] (second and third columns).  I am assuming changing this to [c1] would associate the first column of data with the second y axis.  

The second point of my comment was that doing this in the _Property Editor_ has not effect, which is probably an issue of the Property editor itself, not the chart, but I don't know that for sure.